util/ghidrathon_configure.py

```diff
@@ -91,7 +93,7 @@ def main(args):
     logger.debug('Using Python home located at "%s".', home_path)
 
     json_: str = json.dumps(ghidrathon_save)
-    save_path: pathlib.Path = install_path / "ghidrathon.save"
+    save_path: pathlib.Path = pathlib.Path(os.environ.get(GHIDRATHON_SAVE_PATH, install_path)) / "ghidrathon.save"
```

Comment:
Shoud this path be logged via `logger.debug()` for verbosity?

Reply:
i think that's a good idea